src/server/pkg/metrics/segment.go

```diff
@@ -23,6 +24,28 @@ func reportSegment(metrics *Metrics) {
 		},
 	})
 	if err != nil {
-		lion.Errorf("error reporting to Segment: %s", err.Error())
+		lion.Errorf("error reporting cluster metrics to Segment: %s", err.Error())
+	}
+}
+
+func identifyNewUser(userID string) {
```

Comment:
Yes there is -- its a bit below -- `client.Track()`

This code isn't hooked into anything yet, but in the code where we see a user for the first time we would call this. I think its most likely fine if we call this too aggressively (if they've already been reported), which might make the code simpler